### PR TITLE
Merkle tree operations

### LINKED ISSUE #156: MVP CLI Proposal implementation: Merkle tree opeations
Continuation of implementation of issue #70

- [ ] CLI command to initialise the internal Merkle tree
- [ ] CLI command to set a leaf on the Merkle tree
- [ ] CLI command to set multiple leaves on the Merkle tree
- [ ] CLI command to reset the Merkle tree with multiple values

### PR DESCRIPTION
**Pull request by tyshko-rostyslav**
Closes #156

### CODE REVIEW

**Review by rahulghangas — APPROVED** [review]

File: rln-cli/src/main.rs [hunk 1]
```diff
@@ -51,6 +51,37 @@ fn main() -> Result<()> {
...
+        }
+        Some(Commands::SetMultipleLeaves { index, file }) => {
+            let input_data = File::open(&file)?;
+            state
+                .rln
+                .ok_or(Report::msg("no RLN initialized"))?
+                .set_leaves_from(*index, input_data)?;
+            Ok(())
+        }
+        Some(Commands::ResetMultipleLeaves { file }) => {
```
Comment (rymnc) [inline rln-cli/src/main.rs:77]:
```suggestion
        Some(Commands::InitTree { file }) => {
```

**Review by rymnc — APPROVED** [review]
LGTM, minor nit.
Moving forward with rln-cli PRs, please add @curryrasul as a reviewer too :)

File: rln-cli/src/main.rs (hunk 1 above)
Comment (rymnc) [reply rln-cli/src/main.rs:77]: @tyshko-rostyslav did you see this?

Comment (tyshko-rostyslav) [reply rln-cli/src/main.rs:77]: Sorry, no
Why should we rename it?

**Pull request merged**